.github/workflows/linting.yaml

```diff
@@ -0,0 +1,17 @@
+name: CI Linting
+on: push
```

Comment:
Limit to PRs, or just keep open for any push?

Reply:
Good catch! Made that specific to PRS.